apps/site/src/views/Resources/components/ApiGroup/ApiGroup.tsx

```diff
@@ -0,0 +1,34 @@
+import styles from "./ApiGroup.module.scss";
+import { API_Group_Props } from "../../interfaces/interfaces";
+export default function API_group({
+	title,
+	description,
+	stickerSrc,
+	tagSrc,
+	tagLink,
+	postItSrc,
+}: API_Group_Props) {
+	return (
+		<div
+			style={{
+				backgroundImage: `url(${postItSrc})`,
+				backgroundRepeat: "no-repeat",
+				backgroundSize: "cover",
+				backgroundPositionY: "25px",
+				width: "344px",
+				height: "500px",
```

Comment:
Issue [also for BackendGroup.tsx]: these dimensions does not match the aspect ratio of the background image, so the provided shadow is cut off. We could insert the shadow ourselves and use a colored background instead of an image asset.